zcash_primitives/src/sapling/tree.rs

```diff
@@ -0,0 +1,131 @@
+use bitvec::{order::Lsb0, view::AsBits};
+use group::{ff::PrimeField, Curve};
+use incrementalmerkletree::Altitude;
+use lazy_static::lazy_static;
+use std::io::{self, Read, Write};
+
+use super::{
+    note::Note,
+    pedersen_hash::{pedersen_hash, Personalization},
+};
+use crate::merkle_tree::{HashSer, Hashable};
+
+pub const SAPLING_COMMITMENT_TREE_DEPTH: usize = 32;
+pub const SAPLING_COMMITMENT_TREE_DEPTH_U8: u8 = 32;
```

Comment:
Is this really needed? `SAPLING_COMMITMENT_TREE_DEPTH as u8` [is a const expression](https://doc.rust-lang.org/reference/const_eval.html#constant-expressions).

Reply:
This constant was added by @nuttycom in #417. Nothing in this repo uses it, so I presume it was added for a downstream user, `zcash/zcash` perhaps?